x-pack/plugins/security_solution/public/common/components/exceptions/edit_exception_modal/index.tsx

```diff
@@ -106,8 +108,12 @@ export const EditExceptionModal = memo(function EditExceptionModal({
 
   const onError = useCallback(
     (error) => {
-      addError(error, { title: i18n.EDIT_EXCEPTION_ERROR });
-      onCancel();
+      if (error.message.includes('Conflict')) {
+        setHasVersionConflict(true);
```

Comment:
If there's a conflict error, am showing a EuiCallout and not closing modal. Let's say a user spent 20 minutes writing a comment, for example, if we just close it, they lose it. This gives the user the chance to copy anything they need to.